src/grdfilter.c

```diff
@@ -584,72 +584,79 @@ GMT_LOCAL struct GMT_GRID *init_area_weights (struct GMT_CTRL *GMT, struct GMT_G
+	GMT_Usage (API, 4, "+l Return the lowest mode if multiple modes are found [return average].");
+	GMT_Usage (API, 4, "+u Return the uppermost mode if multiple modes are found [return average].");
+	GMT_Usage (API, 3, "l: Lower : return minimum of all points.");
+	GMT_Usage (API, 3, "L: Lower+ : return minimum of all positive points.");
+	GMT_Usage (API, 3, "m: Median : return the median (50%% quantile) value of all points. "
+		"Append +q<quantile> to select another quantile (in 0-1 range) [0.5].");
+	GMT_Usage (API, 3, "p: Maximum likelihood probability estimator - return mode of all points. "
+		"We approximate the mode using the Least Median of Squares estimator.");
 #if 0
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +c<span> to select a different LMS span, in percent [50].\n");
+	GMT_Usage (API, 4, "+c Select a different LMS <span>, in percent [50].");
 #endif
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +l to return the lowest mode if multiple modes are found [return average].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +u to return the uppermost mode if multiple modes are found [return average].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     u: Upper : return maximum of all points.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     U: Upper- : return maximum of all negative points.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     If <width> is a grid it must match output domain and registration\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-G Set output filename for filtered grid.\n");
+	GMT_Usage (API, 4, "+l Return the lowest mode if multiple modes are found [return average].");
+	GMT_Usage (API, 4, "+u Return the uppermost mode if multiple modes are found [return average].");
+	GMT_Usage (API, 3, "u: Upper : return maximum of all points.");
+	GMT_Usage (API, 3, "U: Upper- : return maximum of all negative points.");
+	GMT_Usage (API, -2, "If <width> is a grid it must match the output domain and registration.");
+	GMT_Usage (API, 1, "\n-G<outgrid>");
+	GMT_Usage (API, -2, "Set output filename for filtered grid.");
 	GMT_Message (API, GMT_TIME_NONE, "\n  OPTIONAL ARGUMENTS:\n");
 #ifdef DEBUG
-	GMT_Message (API, GMT_TIME_NONE, "\t-A DEBUG: Use -A<mode><lon/<lat> to instead save filter specifics at that point. Choose:\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   mode as a (area weights), c (composite weight), r (radii), or w (filter weight).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-A DEBUG: Save area weights to <file> with -As<file> [no save]\n");
+	GMT_Usage (API, 1, "\n-A<mode><lon/<lat>");
+	GMT_Usage (API, -2, "DEBUG: Use -A<mode><lon/<lat> to instead save filter specifics at that point. Choose: "
+		"mode as a (area weights), c (composite weight), r (radii), or w (filter weight).");
+	GMT_Usage (API, 1, "\n-As<file>");
+	GMT_Usage (API, -2, "DEBUG: Save area weights to <file> with -As<file> [no save]");
 #endif
 	GMT_Option (API, "I");
```

Comment:
`+e` and `+n` could need one more indentation level.

![I-option](https://user-images.githubusercontent.com/45796239/123817561-8fbdc600-d8f8-11eb-8b0f-282f3dfd40f9.png)

Reply:
Yes, but already fixed in another PR so not doing it it this one.  All will be well when both are merged.